src/entity-extension.ts

```diff
@@ -13,51 +12,32 @@ import {
 	NodeSpecOverride,
 	omitExtraAttributes,
 	Transaction,
+	extension,
```

Comment:
Is there a good reason to change the order (esp as it's now no longer alphabetically ordered)?